configtx/orderer.go

```diff
@@ -902,6 +899,7 @@ func channelRestrictionsValue(maxChannelCount uint64) *standardConfigValue {
 
 // kafkaBrokersValue returns the config definition for the addresses of the ordering service's Kafka brokers.
 // It is a value for the /Channel/Orderer group.
+// Deprecated: KafkaBrokersValue is deprecated
```

Comment:
// Deprecated: the kafka consensus type is no longer supported.

change all "Deprecated:" comments to this language, there is no need to restate the name of the variable.

Reply:
ok, I changed